home/templatetags/home_tags.py

```diff
@@ -158,3 +158,16 @@ def render_image(context, image, half_width=False, img_class=None):
         'class': img_class,
     })
     return context
+
+
+@register.inclusion_tag('home/tags/meta_tags.html', takes_context=True)
+def render_meta_tags(context):
```

Comment:
can we rename this to 'meta_tags' or 'social_meta_tags'? I want to describe a thing rather than an action.